client/web/wasm/src/wrappers.rs

```diff
@@ -85,8 +85,11 @@ pub fn translate_key(name: &str) -> Key {
 		"r" => KeyR,
 		"m" => KeyM,
 		"x" => KeyX,
```

Comment:
Should we just call a function to lower-case all the strings before they are matched against? This might also be as good a time as ever to add all the letters in the alphabet.